wdl-lint/tests/lints/shellcheck-ok/source.wdl

```diff
@@ -77,3 +77,28 @@ task test3 {
 
     runtime {}
 }
+
+task test4 {
+    meta {}
+
+    parameter_meta {}
+
+    input {
+      Int placeholder
+    }
+
+    command {
+      #@ except: ShellCheck
```

Comment:
Ah sorry, I was unclear. Pulled the syntax from memory instead of looking it up 😅 

I want to make sure that using _the shellcheck directives_ works, as explained on this page - https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-one-specific-instance-in-a-file

So please remove _our directive_ and instead use the SC specific syntax. A little confusing here lol.

Reply:
Ahh that makes more sense. will fix!